src/robot/libraries/Process.py

```diff
@@ -455,6 +457,15 @@ def start_process(
         Earlier versions returned a generic handle and getting the process object
         required using `Get Process Object` separately.
         """
+        # Check for secrets in arguments and unwrap them
+        secret = any(isinstance(arg, Secret) for arg in arguments)
+        if secret:
+            masked_arguments = [
+                "<redacted>" if isinstance(arg, Secret) else arg for arg in arguments
```

Comment:
The `Secret` class itself uses `<secret>` in place of the actual value and for consistency it should be used here as well. An alternative would be using `str(arg)` and use the string representation of the Secret class, but that's currently `Secret(value=<secret>)` which doesn't look good to me in this context. As I just commented on Slack, we could consider changing the string representation, though.